### DISCUSSION TITLE
bi-weekly meeting time

### DISCUSSION THREAD

**Discussion by mflatt** [racket/rhombus#315] [Polls]
The current Rhombus meeting time on Thursdays accommodates East Asia (as Fridays). In particular, given that I run most meetings, it worked for me while on sabbatical in Singapore from last August through May.

The old time had the advantage, for some, of being during regular working hours in the U.S. Since I am now back from sabbatical and in the U.S. again, it works for me to revert to that meeting time.

We could start from scratch to try to pick an optional time for participants around the world, but my guess is that keeping the current time is among our best options. Let's try a poll with simple options.

(Note: Times below shift an hour earlier for some parts of the year in non-U.S. time zones due to daylight saving time. The "Thursday" part is left implicit.)

**Comment by spdegabrielle** [comment]
I'll admit I voted to retain the current time because I feel 4am is too harsh for participants from SDT to attend, and active international participation should be nurtured and supported.

Realistically I am happy to participate asynchronously as my participation is rare to non-existent.

I did like the video! - it is possible to do that more often?

converter that shows daylight/summer times for a selection of timezones: 

https://www.timeanddate.com/worldclock/converter.html?iso=20230607T200000&p1=204&p2=136&p3=224&p4=43&p5=220&p6=64&p7=107&p8=tz_gmt&p9=tz_sgt

**Comment by mflatt** [comment]
At this point, keeping the meeting time has a slight edge, so let's keep it the same for now.

One more alternative to consider: we could alternate meetings between the two times.

**Comment by mflatt** [comment]
As of the July 20, 2023 meeting, we'll try 4:00pm ET (2:00pm MT) again.
